etl/src/interfaces/DatasetInterface.ts

```diff
@@ -5,6 +5,7 @@ export interface StaticMigrable {
   readonly uuid: string;
   readonly table: string;
   readonly year: number;
+  readonly skipStatePersistence: boolean;
```

Comment:
```suggestion
  readonly skipStatePersistence?: boolean;
```

Je pense que c'est plus juste de le mettre en facultatif (je ne crois pas que toutes les classe l'ait, si ?

Reply:
Je l'ai mis dans toutes les classes mais j'ai hésité à faire ce que tu proposes